<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup_user</class>
 <widget class="QDialog" name="signup_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>617</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>User Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>The Registeration Form Of User :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpb_signup">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>560</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpb_back">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grb_inside">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Inside the country</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>43</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="qlb_state">
          <property name="text">
           <string>State :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmb_state">
          <item>
           <property name="text">
            <string>Tehran</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ardabil</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azerbaijan, East</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azerbaijan, West</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bushehr</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chahar Mahaal and Bakhtiari</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fars</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gilan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Golestan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hamadan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hormozgan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ilam</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Isfahan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kerman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kermanshah</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Khorasan, North</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Khorasan, Razavi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Khorasan, South</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Khuzestan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kohgiluyeh and Boyer-Ahmad</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kurdistan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lorestan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Markazi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mazandaran</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qazvin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Semnan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sistan and Baluchestan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alborz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yazd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zanjan</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="qlb_city">
          <property name="text">
           <string>City :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_city">
          <property name="placeholderText">
           <string>Enter Your City</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="qlb_adress">
          <property name="text">
           <string>Adress :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="led_address">
          <property name="placeholderText">
           <string>Enter your Address</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="qlb_postalcode">
          <property name="text">
           <string>Postal code :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="led_postcode">
          <property name="placeholderText">
           <string>Enter Your Postal Code</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>password power :</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlb_password_power">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>421</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="qlb_name">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="qle_name">
        <property name="placeholderText">
         <string>Enter Your Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="qlb_lastname">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Last Name :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="qle_lastname">
        <property name="placeholderText">
         <string>Enter Your Last Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_mobilenumber">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MobileNumber : </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="led_mobilenumber">
        <property name="placeholderText">
         <string>Enter Your MobileNumber</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="qlb_email_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="qle_email">
        <property name="placeholderText">
         <string>Enter Your Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="qlb_username">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>UserName :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="qle_username">
        <property name="placeholderText">
         <string>Enter Your UserName</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="qlb_password">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="qle_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Your Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qpb_show_password1">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="files.qrc">
            <normaloff>:/icon/images/icons8-eye-90.png</normaloff>:/icon/images/icons8-eye-90.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="qlb_password_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Confirm Password :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="qle_password2">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Your Password Again</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qpb_show_password2">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="files.qrc">
            <normaloff>:/icon/images/icons8-eye-90.png</normaloff>:/icon/images/icons8-eye-90.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
